### PR TITLE
feature added 'rm-gitignore'

### PR DESCRIPTION
**Pull request by Sidharth-Singh10**
Fixes #3 
This pull request adds a script that uses `find` to locate all Git repositories within a directory and its subdirectories. It then iterates over each repository, changing into the repository's parent directory, and running `git clean -Xdf` to remove untracked files and directories.

### CODE REVIEW

File: scripts/rm-gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env bash
+git_repos=$(find . -name .git -type d -prune)
+for repo in $git_repos; do
+    cd "$repo/.." 
+    git clean -xdf
```
Comment (rootCircle) [inline scripts/rm-gitignore:4]: You possibly meant `git clean -Xdf` here! git clean -dfx will remove un-ignored files too!

**Review by rootCircle — CHANGES_REQUESTED** [review]

**Comment by rootCircle** [comment]
Congrats on your PR @Sidharth-Singh10, there are some changes required please go through it

ping me here when you're done with those, so we can move forward with your PR!

**Comment by Sidharth-Singh10** [comment]
@rootCircle 
Completed the requested changes.

**Review by rootCircle — APPROVED** [review]

**Comment by rootCircle** [comment]
LGTM :+1: Accepted

**Pull request closed without merging**